models/intel/text-to-speech-en-0001/accuracy-check.yml

```diff
@@ -0,0 +1,31 @@
+evaluations:
+  - name: text-to-speech-en-0001
```

Comment:
model has different name. you need either rename model to this name or refer right model name there